platforms/r3-corda-ent/charts/zone/values.yaml

```diff
@@ -0,0 +1,224 @@
+  volumeSizeZoneH2:
+  # Volume size for logs/ directory
+  # Eg. volumeSizeZoneLogs: 5Gi
+  volumeSizeZoneLogs:
+  # Volume size for data/ directory
+  # Eg. volumeSizeZoneData: 1Gi
+  volumeSizeZoneData:
+podSecurityContext:
+  # Eg. runAsUser: 1000
+  runAsUser:
+  # Eg. runAsGroup: 1000
+  runAsGroup:
+  # Eg. fsGroup: 1000
+  fsGroup:
+
+#############################################################
+#                  CENM Service Details                     #
+#############################################################
+# This section details the CENM service names as per the configuration file
+# It also contains the passwords for keystores and truststores
+cenmServices:
+  # Provide the name of the idman
+  # Eg. idmanName: idman
+  idmanName:
+  # Provide the password of corda-ssl-trust-store.jks
+  # Eg. sslTruststore: password
+  sslTruststore:
+  # Provide the password of corda-ssl-identity-manager-keys.jks
+  # Eg. sslIdmanKeystore: password
+  sslIdmanKeystore:
+
+#############################################################
+#                Database Options and Configuration         #
+#############################################################
+# Database configuration
+database:
+  # Java class name to use for the database
+  # Eg. driverClassName: "org.h2.Driver"
+  driverClassName:
+  # JDBC Driver name
+  # Eg. jdbcDriver: ""
+  jdbcDriver:
+  # The DB connection URL 
+  # Eg. url: "jdbc:h2:file:./h2/zone-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+  url:
+  # DB user
+  # Eg. user: "example-db-user"
+  user:
+  # DB password 
+  # Eg. password: "example-db-password"
+  password:
+  # Migrations of database can be run as part of the startup of Zone, if set to true. 
+  # If set to false, it will be run prior to setting up the Zone.
+  # Eg. runMigration: "true"
+  runMigration:
+
+zoneJar:
+  # value for java -Xmx
+  # Eg. xmx: 1G
+  xmx:
+  # The directory where the Signing Service .jar file is stored
+  # Eg. path: bin
+  path:
+  # The directory where the Signing Service configuration is stored
+  # Eg. configPath: etc
+  configPath:
+
+# Definition of the Auth Service
+authService:
+  # The host path of auth service
+  # Eg. host: auth
+  host:
+  # The port of auth service
+  # Eg. port: 8081
+  port:
+
+# Sleep time (in seconds) after an error occured
+# Eg. sleepTimeAfterError: 120
+sleepTimeAfterError:
+
+# Enable container displaying live logs
+# Eg. logsContainersEnabled: true
+logsContainersEnabled:
+
+########################################
+###   Kubernetes related settings    ###
+########################################
+
+# Name of Kubernetes secret storing credentials to Docker registry
+# Provide the docker-registry secret created and stored in kubernetes cluster as a secret.
+imagePullSecrets:
+  # Eg. name: regcred
+  - name:
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  # Eg. create: false
+  create:
```

Comment:
Don't think this create is used anywhere

Reply:
fixed